### ISSUE TITLE
Registry.createKey or .openKey - cannot read property 'createKey' of undefined

### ISSUE THREAD

**Issue by mehmettoozturk** [simonbuchan/native-reg#27]
Hi ,

My project import native-reg library and using createKey or openKey runtime exception.

import Registry from "native-reg";

export const getRegisterConfig = async () => {
try{
   const regKey = Registry.openKey(Registry.HKCU, "Environment", Registry.Access.ALL_ACCESS);
Registry.closeKey(regKey);
} catch ( error)
{
console.log("error
", error)
}

}

call to function getRegisterConfig();

console log output --> error TypeError: Cannot read property 'openKey' of undefined

thank you

**Comment by mehmettoozturk** [comment]
Hi 

my project ES6 , your library ES5 , can you update ES6?

**Comment by mehmettoozturk** [comment]
Hello there,

Thank you for your support. I tried as you stated and got past the problem.

I have another question. How does the library work?

1step , openKey or createKey
2 step, if we want to change the field we read, key="version" and value = "1" we delete this field with deleteTree.
3 step, create the same field with createKey again
4 step setValueSz  We set Version = "2"
5 step, we closeKey the 3rd step.
6 step, we closeKey the 1st step
